### DISCUSSION TITLE
Field Value from Relationship Picker

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#6176] [Help]
In a form I have one relationship picker for the one side on a one-many and another picker for the many side on a many-many. I added a set value action on another field under the onChange event in an attempt to get the ID field, however the actual value that is entered into the field contains additional characters besides just the ID.

The underlined parts in the respective fields that get filled in are the IDs in this case, but as can be seen by the screen shot there are additional characters.

![image](https://user-images.githubusercontent.com/106758851/171692924-1ba2524b-6787-4ab2-a41d-0ead310d52e8.png)

**Comment by melohagan** [comment]
Hi @mmalessa80 

It appears to be encoding the references like: `['<ref>']` 

I've raised a bug from this discussion: https://github.com/Budibase/budibase/issues/6194
